multicluster/controllers/multicluster/member_clusterset_controller.go

```diff
@@ -132,7 +132,7 @@ func (r *MemberClusterSetReconciler) SetupWithManager(mgr ctrl.Manager) error {
 
 func (r *MemberClusterSetReconciler) updateMultiClusterSetOnMemberCluster(clusterSet *multiclusterv1alpha1.ClusterSet) error {
 	if r.RemoteCommonAreaManager == nil {
-		r.RemoteCommonAreaManager = commonarea.NewRemoteCommonAreaManager(r.clusterSetID, r.clusterID)
+		r.RemoteCommonAreaManager = commonarea.NewRemoteCommonAreaManager(r.clusterSetID, r.clusterID, r.Namespace)
```

Comment:
I do not get where you lock the mutex?

Reply:
the whole reconciling process is locked https://github.com/antrea-io/antrea/blob/2065919367e399256dc5b10db501080e21f8568f/multicluster/controllers/multicluster/member_clusterset_controller.go#L67-L68

Reply:
I actually wanted to ask about GatewayReconciler and StaleReconciler - I do not see we lock their mutexes for write.